File: Week_2/problem_6-Summation.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/A

n = int(input())
ar = list(map(int, input().split()))

sum_val = sum(ar)

# Convert sum to its absolute value if negative
if sum_val < 0:
    sum_val = abs(sum_val)

print(sum_val)

File: Week_2/problem_6-Summation.c
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/A

#include <stdio.h>

int main()
{
    int n;
    long long int sum = 0;

    scanf("%d", &n);
    int ar[n];

    for (int i = 0; i < n; i++)
    {
        scanf("%d ", &ar[i]);
        // printf("%d\n", ar[i]);

        sum = sum + ar[i];
    }

    if (sum < 0)
    {
        // to convert our sum to absolute sum
        sum = sum * -1;
    }
    printf("%lld", sum);

    return 0;
}
